### ISSUE TITLE
lv_list not scrolling with LV_KEY_PREV and LV_KEY_NEXT

### ISSUE THREAD

**Issue by jkent** [lvgl/lvgl#2060]
- [x] Check the related part of the [Documentation](https://docs.lvgl.io/)
- [x] Update lvgl to the latest version
- [x] Reproduce the issue in a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html)

### Describe the bug

lv_list is not scrolling to the selected item when using LV_INDEV_TYPE_KEYPAD and LV_KEY_PREV and LV_KEY_NEXT.  Selection is off screen

### To Reproduce
```c
void demo(void)
{
    lv_group_t* g = lv_group_create();

    lv_indev_drv_t kb_drv;
    lv_indev_drv_init(&kb_drv);
    kb_drv.type = LV_INDEV_TYPE_KEYPAD;
    kb_drv.read_cb = keyboard_read;
    lv_indev_t * kb_indev = lv_indev_drv_register(&kb_drv);
    lv_indev_set_group(kb_indev, g);

    lv_obj_t* list = lv_list_create(lv_scr_act(), NULL);
    lv_obj_set_size(list, LV_HOR_RES, LV_VER_RES);

    char text[16];
    for (int i = 0; i < 26; i++) {
        sprintf(text, "Item %c", 'A' + i);
        lv_obj_t* list_btn = lv_list_add_btn(list, NULL, text);
        lv_group_add_obj(g, list_btn);
    }
}
```

### Expected behavior
Tabbing through the list should result in the list being scrolled, I.E. "Item I" being shown once it is selected.

A little off topic, but I think the way the arrow keys work with LV_INDEV_TYPE_KEYPAD is a little broken.  On screen keyboard cannot be used with such a setup.

### Screenshots or video
![Screenshot at 2021-02-07 19-33-59](https://user-images.githubusercontent.com/556979/107166390-76cd6c00-697b-11eb-8050-fe1c1fe8e5ed.png)

Even better, this repository has a simulator setup that shows the problem, using the Up/Down arrow keys:  `https://gitlab.jkent.net/odroid-go/golauncher`

Be sure to --recurse-submodules !

Just `cd` to simulator and `make run`

**Comment by embeddedt** [comment]
Fix proposed in #2061. Thanks for the report!

By the way, I had to apply the following patch to your Makefile to get it to build, as GCC normally requires libraries to be specified after object files, not before:

```diff
diff --git a/simulator/Makefile b/simulator/Makefile
index 47e998d..52129da 100644
--- a/simulator/Makefile
+++ b/simulator/Makefile
@@ -183,7 +183,7 @@ $(BUILD_DIR)/%.o: %.cpp
 $(TARGET): $(OBJECTS)
        @echo "  LD      $@"
        @mkdir -p $(BUILD_DIR)
-       @$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
+       @$(CC) $(CFLAGS) -o $@ $^ $(LDFLAGS)
 
 run: $(TARGET)
        @$(TARGET)
```

**Comment by jkent** [comment]
Thanks! and thanks for the patch!
